src/gnatss/loaders.py

```diff
@@ -176,32 +178,59 @@ def load_roll_pitch_heading(files: List[str]) -> pd.DataFrame:
 
     Parameters
     ----------
-    files : List[str]
-        The list of path string to files to load
+    files : Union[List[str], str]
+        The list of path string to files to load, or an empty string if no file paths found
 
     Returns
     -------
     pd.DataFrame
-        Pandas DataFrame containing all of
-        the roll pitch heading data.
-        Expected columns will have 'time' and
-        the 'roll', 'pitch', 'heading' values
+        Pandas DataFrame containing all of the roll pitch heading data.
+        Expected columns will have 'time' and the 'roll', 'pitch', 'heading' values.
+        Return empty Dataframe if files is empty string.
     """
     columns = [
         constants.RPH_TIME,
-        constants.RPH_ROLL,
-        constants.RPH_PITCH,
-        constants.RPH_HEADING,
-    ]
-    # Read all rph files
-    rph_dfs = [
-        pd.read_csv(i, delim_whitespace=True, header=None, names=columns)
-        .drop_duplicates(constants.RPH_TIME)
-        .reset_index(drop=True)
-        for i in files
+        *constants.RPH_COLUMNS,
     ]
-    all_rph = pd.concat(rph_dfs).reset_index(drop=True)
-    return all_rph
+    if files:
+        # Read all rph files
+        rph_dfs = [
+            pd.read_csv(i, delim_whitespace=True, header=None, names=columns)
+            .drop_duplicates(constants.RPH_TIME)
+            .reset_index(drop=True)
+            for i in files
+        ]
+        all_rph = pd.concat(rph_dfs).reset_index(drop=True)
+        return all_rph
+    else:
+        return pd.DataFrame(columns=columns)
+
+
+def load_atd_offsets(config: Configuration) -> Union[NDArray[Shape["3"], Float], None]:
```

Comment:
```suggestion
def get_atd_offsets(config: Configuration) -> Union[NDArray[Shape["3"], Float], None]:
```
Since this is technically not I/O operation, let's change the function name to be "get" since we are getting the values from config